### PR TITLE
Mobile geoeditor configurability

### PR DESCRIPTION
**Pull request by simonseyock**
Makes geoeditor configurable through default config.

For this the config was moved out of `componentes.geoeditor` to the root. This means that **ALL** applications using geoeditor need to be adjusted accordingly.

```yaml
components:
  geoeditor:
    config: 'value'
```

becomes
```yaml
components:
  geoeditor: true

geoeditor:
  config: 'value'
```

### CODE REVIEW

File: munimap/static/js/draw-layer-config.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ angular.module('munimapDraw', [
         function(LayersService, DrawService, DefaultStyle, ColorpickerPalette, munimapConfig,
             StyleSchema, StyleSchemaFormOptions, LabelSchema, LabelSchemaFormOptions, DrawIconsPrefix) {
 
-            if(munimapConfig.components.draw !== undefined || munimapConfig.components.geoeditor !== undefined) {
+            if(munimapConfig.components.draw !== undefined || munimapConfig.components.geoeditor === true) {
```
Comment (Unraveler) [inline munimap/static/js/draw-layer-config.js:58]: why not simply `!!munimapConfig.components.draw || !!munimapConfig.components.geoeditor`

File: munimap/translations/messages.pot [hunk 2]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2021-06-29 16:33+0200\n"
+"POT-Creation-Date: 2021-07-01 13:59+0200\n"
```
Comment (Unraveler) [inline munimap/translations/messages.pot:11]: Just a question, but shouldn't a date not be hardcoded, or it is just a placeholder?

**Review by Unraveler — APPROVED** [review]
JUst two minor comments. Otherwise looks good

File: munimap/static/js/draw-layer-config.js (hunk 1 above)
Comment (simonseyock) [reply munimap/static/js/draw-layer-config.js:58]: don't know. the other version just a little more verbose.

File: munimap/translations/messages.pot (hunk 2 above)
Comment (simonseyock) [reply munimap/translations/messages.pot:11]: The `messages.pot` and `messages.po` files are auto-generated and edited by hand. This just marks when it happened the last time.

File: munimap/static/js/draw-layer-config.js (hunk 1 above)
Comment (simonseyock) [reply munimap/static/js/draw-layer-config.js:58]:
```suggestion
            if(!!munimapConfig.components.draw || !!munimapConfig.components.geoeditor) {
```

**Pull request merged**